app.js

```diff
@@ -22,5 +23,36 @@ function handleError(error) {
     console.warn(`Error displaying position, error: ${error.code}`)
 }
 
-navigator.geolocation.getCurrentPosition(printLocation, handleError);
+navigator.geolocation.getCurrentPosition(getLocation, handleError);
+
+// Getting the temperature from the Open Weather API using geographical coordinates.
+// Response format is Json by default.
+// https://openweathermap.org/current --free
+
+const KEY = "3137461397c70af0e13c77fda97afa11"
+
+function getTemp(latitude, longitude){
+    let api = `http://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${KEY}&units=metric`
+    //the fetch() method instructs the web browsers to send a request to a URL.
+    //less than60 calls/min
+    fetch(api)
+    .then(response => {
+        // handles response
+        // console.log(response.statusText); when response.status is 200, it is OK.
+        if (response.status != 200){
+            console.log("there is a problem with the request")
+        }else{
+            // data --> Promise object represents the eventual completion (or failure) 
+            // of an asynchronous operation and its resulting value.
+            let data = response.json().then(data=> {
```

Comment:
Since you already using fetch and api I'll suggest you to look at this article https://javascript.info/promise-chaining
This will give you understanding that you can rewrite this function a bit differently. So you won't need to do the `response.json()` on that line.
It will start like that:
```js
fetch(api)
.then(response => {
  return response.json();
})
.then(data => ...)
.catch();
```

Reply:
changed :)